README.md

```diff
@@ -97,35 +103,36 @@ By the end of this lesson. You should be able to set up two separate servers tha
 * **Question:** What is happening on the first couple of lines of the new file you've created?
 
 * **Your Answer:** 
-
+Creating the local environment variable
 ---
 
 - [ ] Let's store the token in LocalStorage with a key of `journal-app`.
 
 * **Question:** Why are we storing the token?
+So we can ask for it to protect access to logged-in only features
 
 * **Your Answer:**
-
```

Comment:
Easier to use, also so we maintain a logged in state when user refreshes or manually visits a route in browser.